src/i18n/locales/en/translation.json

```diff
@@ -247,6 +247,16 @@
     "text_stopping": "Stopping",
     "button_show_report": "Show earnings report",
     "button_show_orderbook": "Show orderbook",
+    "current": {
+      "title": "Active Offer #{{ id }}",
+      "text_nickname": "Nym",
```

Comment:
I'd go with Offer ID or just ID here, what do you think?

If needed we could also do `<nym>:<oid>` to make it a fully unique ID? 🤔

Reply:
Opted for just showing `<nym>:<oid>` :+1: